test-utils/runtime-tester/run_scenario/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use clap::{App, Arg};
+use runtime_tester::Scenario;
+
+use std::path::Path;
+use std::time::{Duration, Instant};
+
+fn main() {
```

Comment:
Is this just to reproduce a failure? `cargo fuzz` has a built-in feature for this (when a case fails, it prints the command you need to run to re-run this single case). I would probably default to using that instead of our own tool, but no strong opinion!